sites/platform/src/create-apps/app-reference.md

```diff
@@ -375,31 +412,122 @@ applications:
 ```
 {{% /version/specific %}}
 
-For examples of how to set up a `service` mount, see the dedicated [Network Storage page](/add-services/network-storage.md).
+### Ensure continuity when changing the name of your mount
 
-The accessibility to the web of a mounted directory depends on the [`web.locations` configuration](#web).
-Files can be all public, all private, or with different rules for different paths and file types.
+Changing the name of your mount affects the default `source_path`.
 
-Note that when you back up an environment,
-the mounts on that environment are backed up too.
+Say you have a `/web/uploads/` mount with an undefined `source_path`:
 
-Also, mounted directories aren't deleted when they're removed from `{{< vendor/configfile "app" >}}`.
-The files still exist on disk until manually removed (or, for `tmp` mounts, until the app container is moved to another host during a maintenance operation).
+```yaml {configFile="app"}
+mounts:
+    '/web/uploads/':
+        source: local
+```
+
+ If you rename the mount to `/my/uploads/`, it will point to a new, empty `/my/uploads/` directory.
+ 
+ To ensure continuity, you need to explicitly define the `source_path` as the previous name of the mount, without leading or trailing slashes:
+ 
+ ```yaml {configFile="app"}
+mounts:
+    '/my/uploads/':
+        source: local
+        source_path: web/uploads
+```
+ 
+The `/my/uploads/` mount will point to the original `/web/uploads/` directory, maintaining access to all your existing files in that directory.
```

Comment:
Is this chunk for PSH only, or also for Upsun?
Asking because I don't see the `<!-- Platform.sh -->` tag, and it mentions `local` mount which doesn't exist on Upsun.

Reply:
My bad, changed it to the `tmp` type since it's supported by both products.